meshroom/ui/qml/GraphEditor/NodeEditor.qml

```diff
@@ -24,6 +24,92 @@ Panel {
     icon: MaterialLabel { text: MaterialIcons.tune }
 
     headerBar: RowLayout {
+        Label {
+            text: {
+                if (node !== null && node.isSubmittedOrRunning()) {
+                    // Some chunks might be submitted but they'll all run eventually
+                    if (node.elapsedTime > 0) { // At least a chunk is done running
+                        return "Running for: " + getTimeStr(node.elapsedTime)
+                    } else {
+                        return (node.chunks.count > 1) ? "First chunk running" : "Node running"
+                    }
+                } else if (node !== null && node.isFinishedOrRunning()) {
+                    /* Either all chunks finished running or the last one is running
+                        * Placed inside an "else if" instead of "else" to avoid entering the functions
+                        * when there is no real use */
+                    return getTimeStr(node.elapsedTime)
+                } else {
+                    return ""
+                }
+            }
+            padding: 2
+            font.italic: true
+            visible: {
+                if (node !== null) {
+                    if ((node.isFinishedOrRunning() || node.isSubmittedOrRunning())) {
+                        return true
+                    }
+                }
+                return false
+            }
+
+            ToolTip.text: {
+                if (node !== null && (node.isFinishedOrRunning() || (node.isSubmittedOrRunning() && node.elapsedTime > 0))) {
+                    var longestChunkTime = getLongestChunkTime(node.chunks)
+                    if (longestChunkTime > 0)
+                        return "(longest chunk: " + getTimeStr(longestChunkTime) + ")"
+                    else
+                        return ""
+                } else {
+                    return ""
+                }
+            }
+            ToolTip.visible: ToolTip.text ? runningTimeMa.containsMouse : false
+            MouseArea {
+                id: runningTimeMa
+                anchors.fill: parent
+                hoverEnabled: true
+            }
+
+            function getTimeStr(elapsed)
+            {
+                if (elapsed <= 0)
+                    return ""
+
+                var hours = 0
+                var min = 0
+                var sec = 0
+                var finalTime = ""
+
+                if (elapsed > 3600) {
+                    hours = Math.floor(elapsed / 3600)
+                    elapsed = elapsed - (hours * 3600)
+                    finalTime += hours + "h"
+                }
+                if (elapsed > 60) {
+                    min = Math.floor(elapsed / 60)
+                    elapsed = elapsed - (min * 60)
+                    finalTime += min + "m"
+                }
+
+                finalTime += Number(elapsed.toLocaleString(Qt.locale('en-US'))) + "s"
```

Comment:
Need a round to convert to interger.

"1h52m3.12s" is quite difficult to read (not coherent int/float between units)
And second precision is enough, so we should move to "1h52m3s".